src/net-questdb-client/Ingress/Buffers/BufferStreamContent.cs

```diff
@@ -0,0 +1,81 @@
+using System.Net;
+using QuestDB.Ingress.Enums;
+using QuestDB.Ingress.Utils;
+
+namespace QuestDB.Ingress.Buffers;
+
+public class BufferStreamContent : HttpContent
+{
+    public BufferStreamContent(Buffer buffer)
+    {
+        Buffer = buffer;
+    }
+
+    private Buffer Buffer { get; }
+
+    /// <summary>
+    ///     Writes the chunked buffer contents to a stream.
+    ///     Used to fulfill the <see cref="HttpContent" /> requirements.
+    /// </summary>
+    /// <param name="stream"></param>
+    /// <param name="context"></param>
+    /// <exception cref="IngressError">When writing to stream fails.</exception>
+    protected override async Task SerializeToStreamAsync(Stream stream, TransportContext? context)
```

Comment:
Please add the method `Buffer.writeToStreamAsync(Stream stream)` and call it from here. Make all Buffer fields private do not expose Buffer.Buffers.